### PR TITLE
update coq-elpi dev file

### PR DESCRIPTION
**Pull request by gares**

### CODE REVIEW

**Comment by gares** [comment]
@palmskog the file was terribly outdated.

File: extra-dev/packages/coq-elpi/coq-elpi.dev/opam [hunk 1]
```diff
@@ -1,38 +1,44 @@
...
-
 depends: [
-  "ocaml" {>= "4.10.0"}
+  "dune" {>= "3.13"}
+  "ocaml" {>= "4.09.0"}
   "stdlib-shims"
-  "elpi" {= "1.19.2" }
-  "coq" {= "dev"}
+  "elpi" {>= "1.18.2" & < "1.20.0~"}
+  "coq" {= dev}
```
Comment (palmskog) [inline extra-dev/packages/coq-elpi/coq-elpi.dev/opam:22]:
```suggestion
  "coq" {= "dev"}
```

**Comment by palmskog** [comment]
cc: @erikmd after this PR is merged, I think the MathComp Docker image problems with `coq-elpi.dev` and `coq-hierarchy-builder.dev` will be solved

File: extra-dev/packages/coq-elpi/coq-elpi.dev/opam (hunk 1 above)
Comment (erikmd) [inline extra-dev/packages/coq-elpi/coq-elpi.dev/opam:22]: shouldn't this be `  "coq" {>= "8.19"}` like https://github.com/LPCIC/coq-elpi/blob/master/coq-elpi.opam ?

If yes, that would be enough given `"dev" >= "8.19"`

Comment (gares) [reply extra-dev/packages/coq-elpi/coq-elpi.dev/opam]: sincerly I don't know what is best... I'm fine with any solution that works ;-)

**Review by erikmd — APPROVED** [review]
> sincerly I don't know what is best... I'm fine with any solution that works ;-)

Both are OK for coq.dev; the only question IMHO was:

is coq-elpi.dev required for compiling math-comp.dev with coq 8.19 (< dev) ?

according to [this job](https://gitlab.inria.fr/math-comp/math-comp/-/jobs/4489478#L1653), it seems it is not required.

So I approve this PR as is

**Pull request merged**